.jenkins/pytorch/win-test-helpers/test_python_jit_legacy.bat

```diff
@@ -3,7 +3,7 @@ call %SCRIPT_HELPERS_DIR%\setup_pytorch_env.bat
 pushd test
 
 echo Run jit_profiling tests
-python run_test.py --include test_jit_profiling test_jit_fuser_te test_tensorexpr --verbose --determine-from="%1"
+python run_test.py --include test_jit_legacy test_jit_fuser_legacy --verbose --determine-from="%1"
```

Comment:
Those are already covered in, aren't they?
https://github.com/pytorch/pytorch/blob/77cd8e006b92a8faf90a1ffcfe601860df587726/.jenkins/pytorch/win-test-helpers/test_python_all_except_nn.bat#L2